py3status/module.py

```diff
@@ -80,6 +82,17 @@ def clear_cache(self):
             if self.config['debug']:
                 syslog(LOG_INFO, 'clearing cache for method {}'.format(meth))
 
+    def set_updated(self):
+        """
+        Mark the module as updated
+        """
+        self.py3_wrapper.notify_update(self.module_full_name)
+
+    def get_latest(self):
+        for method in self.methods.values():
+            output = method['last_output']
+        return output
```

Comment:
ahh,

I thought it did that but looking at the code it looks like modules can have multiple responses showing. I'll make the changes for that when I get the chance.

You are very thorough with you code reviews, which is great!